tests/fuzzing/test_fuzz_core.py

```diff
@@ -233,9 +235,14 @@ async def _fuzzer_cmd(id, core, workspace, fs_state):
 
 @pytest.mark.trio
 @pytest.mark.slow
-async def test_fuzz_core(request, running_backend, alice_core):
-    await trio.sleep(0.1)  # Somehow fixes the test
-    wid = await alice_core.user_fs.workspace_create(EntryName("w"))
+async def test_fuzz_core(request, running_backend, alice_core: LoggedCore):
+    # FIXME: Does this "power nap" is really required ?
+    # await trio.sleep(0.1)  # Somehow fixes the test
+    with trio.fail_after(1):
+        await alice_core.wait_idle_monitors()
+    with trio.fail_after(1):
+        wid = await alice_core.user_fs.workspace_create(EntryName("w"))
```

Comment:
This line will trigger the sync monitor, so shouldn't we wait for the idle monitors after that?

Reply:
Maybe, but I find it strange that we directly use `user_fs` instead of `LoggedCore` to create the workspace.

I could add a `wait_idle_monitors` after the workspace create.

Reply:
I've tried adding a `wait_idle_monitors` after the workspace creation.

It result in `TooSlowError`.